rest/taskana-rest-spring/src/test/java/pro/taskana/classification/rest/ClassificationControllerRestDocumentationTest.java

```diff
@@ -0,0 +1,95 @@
+package pro.taskana.classification.rest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.delete;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.get;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.post;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.put;
+
+import java.util.List;
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import pro.taskana.classification.api.ClassificationService;
+import pro.taskana.classification.api.models.Classification;
+import pro.taskana.classification.api.models.ClassificationSummary;
+import pro.taskana.classification.rest.assembler.ClassificationRepresentationModelAssembler;
+import pro.taskana.classification.rest.models.ClassificationRepresentationModel;
+import pro.taskana.common.rest.RestEndpoints;
+import pro.taskana.common.test.BaseRestDocumentationTest;
+
+@TestMethodOrder(OrderAnnotation.class)
+class ClassificationControllerRestDocumentationTest extends BaseRestDocumentationTest {
+
+  @Autowired ClassificationRepresentationModelAssembler assembler;
+  @Autowired ClassificationService classificationService;
+
+  @Test
+  void getAllClassificationsDocTest() throws Exception {
+    mockMvc
+        .perform(get(RestEndpoints.URL_CLASSIFICATIONS))
+        .andExpect(MockMvcResultMatchers.status().isOk());
+  }
+
+  @Test
+  void getClassificationDocTest() throws Exception {
+    mockMvc
+        .perform(
+            get(RestEndpoints.URL_CLASSIFICATIONS_ID, "CLI:100000000000000000000000000000000009"))
+        .andExpect(MockMvcResultMatchers.status().isOk());
+  }
+
+  @Test
+  @Order(1)
+  void createClassificationDocTest() throws Exception {
+    Classification classification =
+        classificationService.newClassification("Key0815casdgdgh", "DOMAIN_B", "TASK");
+    ClassificationRepresentationModel repModel = assembler.toModel(classification);
+    mockMvc
+        .perform(
+            post(RestEndpoints.URL_CLASSIFICATIONS)
+                .content(objectMapper.writeValueAsString(repModel)))
+        .andExpect(MockMvcResultMatchers.status().isCreated());
+  }
+
+  /**
+   * this test only works with the {@link
+   * ClassificationControllerRestDocumentationTest#createClassificationDocTest()} test.
```

Comment:
Why did we add this dependency between these 2 tests? Couldn't we delete another classification here?

Reply:
true.